### PR TITLE
Backmerge tcgc(0.44.2) and autorest (0.44.1) hotfix

### PR DESCRIPTION
**Pull request by tadelesh**

### CODE REVIEW

File: pnpm-lock.yaml
Comment (timotheeguerin) [inline pnpm-lock.yaml:1]: I don't think supposed to be that many change to that file here, can you try reseting to the main version and running `pnpm install`

Comment (tadelesh) [reply pnpm-lock.yaml:1]: i've rerun the `pnpm install` but still many changes. i don't know if it comes from add openapi dep in tcgc. i've compared other files in tcgc and all are good compared with the release branch.

Comment (tadelesh) [reply pnpm-lock.yaml:1]: @timotheeguerin could you help to merge this?

Comment (timotheeguerin) [reply pnpm-lock.yaml:1]: Did you reset first from the version from main

Comment (timotheeguerin) [reply pnpm-lock.yaml:1]: It's something we have to be careful as it likely revert the recent upgrade of dependencies

Comment (tadelesh) [reply pnpm-lock.yaml:1]: resolved.

**Review by timotheeguerin — APPROVED** [review]

**Pull request merged**